pyatlan/model/search.py

```diff
@@ -370,7 +371,9 @@ def with_created_by(cls, value: StrictStr):
 
     @classmethod
     @validate_arguments()
-    def with_glossary(cls, value: StrictStr):
+    def with_glossary(
+        cls, value: constr(strip_whitespace=True, min_length=1, strict=True)  # type: ignore
```

Comment:
Should we rename the `value` argument to `qualified_name` (or `name`)? To make it clear to someone using the method which input they're expected to provide (`value` is too generic, I think?)

Reply:
Done.